witness/tests/Test/Dscp/Witness/Explorer/ExplorerSpec.hs

```diff
@@ -73,9 +76,27 @@ getTransactionsPaged chunkSize mAddress = getFrom Nothing
         return (plItems txList : next)
 
 spec :: Spec
-spec = describe "Explorer" $ do
+spec = describe "Explorer" . divideMaxSuccessBy 10 $ do
```

Comment:
Does this test takes too much time? Are you sure that coverage isn't decreased drastically due to this?

Reply:
Before the change, they took `55 sec` to complete (now `3.5 sec`). I found that a lot as soon as this is far not a complete list of required tests for the explorer.
IMO Coverage should be enough for now, all current tests cases have just two-three qualitatively different situations (e.g. 1 tx / many txs; `since` parameter refers to chain / to mempool).